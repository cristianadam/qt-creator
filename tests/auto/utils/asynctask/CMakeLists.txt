add_qtc_test(tst_utils_asynctask
  DEPENDS Utils
  SOURCES tst_asynctask.cpp
)
