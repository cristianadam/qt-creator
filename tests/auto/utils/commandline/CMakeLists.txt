add_qtc_test(tst_utils_commandline
  DEPENDS Utils app_version
  SOURCES tst_commandline.cpp
)
