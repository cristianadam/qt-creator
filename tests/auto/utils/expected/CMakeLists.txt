add_qtc_test(tst_utils_expected
  DEPENDS Utils
  SOURCES tst_expected.cpp
)
